shipyard/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
You can remove this file I think!

Reply:
Since this tile includes an integration but doesn't send in any metrics, this file is needed, but can be left blank